source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -431,20 +430,20 @@ export class CardDesigner extends Designer.DesignContext {
         this.recreateDesignerSurface();
 
         if (this._deviceEmulationChoicePicker) {
-            this._deviceEmulationChoicePicker.isEnabled = !!this.hostContainer.enableDeviceEmulation;
+            this._deviceEmulationChoicePicker.isHidden = !(!!this.hostContainer.enableDeviceEmulation);
```

Comment:
that's a good point. Can customers create custom containers?

The only reason I could think of needing `!!` is if a custom implementation returned `null` for `enableDeviceEmulation`, but would that matter since null evaluates to falsey?